src/components/listing/Slider/index.jsx

```diff
@@ -51,7 +50,7 @@ function PrevArrow({ onClick }) {
  * Listing page slider section.
  *
  * @param {Object} props        The component props.
- * @param {Object} props.images The component data.
+ * @param {Object} props.images Listing page slider images.
```

Comment:
```suggestion
 * @param {Object[]} props.images Listing page slider images.
```

This should be an array of objects